articles/user-guide/libraries/additional-libraries.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Using additional Q# libraries
+description: Learn how to add additional Q# libraries to your quantum applications.
+author: cgranade
+ms.author: chgranad
+ms.date: 06/30/2020
+ms.topic: article
+uid: microsoft.quantum.libraries.using
+---
+
+# Using Additional Q# Libraries
+
+The Quantum Development Kit provides additional domain-specific functionality through _NuGet packages_ that can be added to your Q# projects.
+
+| Q# Library  | NuGet package | Notes |
+|---------|---------|--------|
+| [Q# standard library](xref:microsoft.quantum.libraries.standard.intro) | [**Microsoft.Quantum.Standard**](https://www.nuget.org/packages/Microsoft.Quantum.Standard) | Included by default |
+| [Quantum chemistry library](xref:microsoft.quantum.chemistry.concepts.intro) | [**Microsoft.Quantum.Chemistry**](https://www.nuget.org/packages/Microsoft.Quantum.Chemistry) | |
+| [Quantum numerics library](xref:microsoft.quantum.numerics.intro) | [**Microsoft.Quantum.Numerics**](https://www.nuget.org/packages/Microsoft.Quantum.Numerics) | |
+| [Quantum machine learning library](xref:microsoft.quantum.libraries.machine-learning.intro) | [**Microsoft.Quantum.MachineLearning**](https://www.nuget.org/packages/Microsoft.Quantum.MachineLearning) | |
+
+Once you have installed the Quantum Development Kit for use with your preferred environment and host language, you can easily add libraries to individual Q# projects without any further installation.
+
+> [!NOTE]
+> Some Q# libraries may work well with additional tools that work alongside your Q# programs, or that integrate with your host applications.
+> For example, the [chemistry library installation instructions](xref:microsoft.quantum.chemistry.concepts.installation) describe how to use the [**Microsoft.Quantum.Chemistry** package](https://www.nuget.org/packages/Microsoft.Quantum.Chemistry) together with the NWChem computational chemistry platform, and how to install the `qdk-chem` command-line tools for working with quantum chemistry data.
+
+## [Q# command-line applications or .NET interopability](#tab/tabid-csproj)
+
+**Command line or Visual Studio Code:** Using the command line on its own or from within Visual Studio Code, you can use the `dotnet` command to add a NuGet package reference to your project.
+For example, to add the [**Microsoft.Quantum.Numerics**](https://www.nuget.org/packages/Microsoft.Quantum.Numerics) package, run the following command:
+
```

Comment:
I think it might be beneficial to highlight a little more that the libraries are added for the _specific_ projects, and not system-wide. In doing so, I'd say it's also worth noting that the `dotnet add package ...` command should be run from the location of the specific Q# project

Reply:
Agreed, thanks for raising that. A large part of my goal here was to get rid of any discussion of "installing" a library, since that was generating a lot of confusion between project-local and user-local packages. I'll revise accordingly, as per your suggestion.

Reply:
Couldn't agree more, that confusion was very much being caused by the old version (I myself thought for a long time that it was something that needing installation). Currently writing the MS Learn module on libraries and the preview of this PR was already hugely helpful there